pages/common/mount.md

```diff
@@ -21,3 +21,7 @@
 - Mount a specific filesystem described in /etc/fstab (e.g. "/dev/sda1 /my_drive ext2 defaults 0 2"):
 
 `mount {{/my_drive}}`
+
+- Mount a directory to another directory (bind mount) \{ --bind || -B \} {{olddir}} {{newdir}}:
+
+`mount --bind {{/tmp}} {{/usr/tmp}}`
```

Comment:
I think you can write placeholder dirs here itself to make it simple and clear. Concrete examples here does not make it clear which is the olddir and which is the newdir. Please write it in the form of {{path/to/old_dir}}.

And we can also remove "(bind mount) \{ --bind || -B \}"

Reply:
How's that look?

Reply:
The directory values should be path/to/old_dir and path/to/new_dir respectively